Makefile

```diff
@@ -1,27 +1,55 @@
 CLI_DIR:=$(CURDIR)/components/cli
 ENGINE_DIR:=$(CURDIR)/components/engine
 PACKAGING_DIR:=$(CURDIR)/components/packaging
+MOBY_COMPONENTS_URL=https://raw.githubusercontent.com/shykes/moby-extras/master/cmd/moby-components
```

Comment:
let's lock this so it is explicit what code is run: https://raw.githubusercontent.com/shykes/moby-extras/c54159187efebc6155821dc0493c2294e83203c1/cmd/moby-components

Reply:
Updated to include SHA as a variable to make it easier to change later on.